api/apps/api/test/utils/test-client/entity-seeder.ts

```diff
@@ -0,0 +1,39 @@
+import { DataSource } from 'typeorm';
+import { apiConnections } from '@marxan-api/ormconfig';
+
+/**
+ * @description Utility function to create entities in the DB avoiding hitting the test API endpoints
```

Comment:
nice. With some "small" ergonomics improvements it could become a very useful (and kinda dangerously sharp) utility too.

Ergonomics improvements: I'd suggest mainly to avoid having to manually create dependencies and pass them on to the `seed()` call for dependent entities, which is a bit like juggling with sharp tools while hopping on a yoga ball.

But I do definitely see the place for something like this in use cases like the temporary test suite in this PR - well done!

Reply:
Hm, I am not sure what you mean by ` avoid having to manually create dependencies and pass them`, some entities need to have some dependencies due to DB constraints (and some of them recursively).

If you mean encapsulating the creation of the dependencies to happen under the hood, I don't see how we can do that since this is a generic class. Unless we go for a utility for each entity.

And then, I didn't go for this option as I am still not sure how the data model works. Actually this was a great exercise to start understanding it!

Reply:
yep, I was getting ahead of ourselves - I value the approach of validating db constraints this way, and if we need more of this kind of tests, we can find a way to make the approach feel more DSL-like and (to get, for example, a fully formed set of organization/user/project - for example, here - so that we can directly use the project when exercising the SUT, in this case the cost surface creation).

Basically instead of creating org/user/project in the tests before creating a cost surface, again following the example of these tests, we'd "just" create the project _through a utility function in the test itself_, and let the utility function encapsulate the creation of org and user needed for the project, using the same generic class under the hood.